### ISSUE TITLE
Import to Intune not working due to invalid argument

### ISSUE THREAD

**Issue by doe81** [aaronparker/vcredist#132]
# Expected Behavior

Import Vc to intune

# Current Behavior

Not working due to invalid argument.

# Failure Information (for bugs)

```
VERBOSE: Populating RepositorySourceLocation property for module IntuneWin32App.
VERBOSE: Loading module from path 'C:\Program Files\WindowsPowerShell\Modules\IntuneWin32App\1.4.1\IntuneWin32App.psm1'.
VERBOSE: Support module installed: IntuneWin32App.
New-IntuneWin32AppRequirementRule : Cannot validate argument on parameter 'MinimumSupportedWindowsRelease'. The argument "1809" does not belong
to the set "W10_1607,W10_1703,W10_1709,W10_1803,W10_1809,W10_1903,W10_1909,W10_2004,W10_20H2,W10_21H1,W10_21H2,W10_22H2,W11_21H2,W11_22H2"
specified by the ValidateSet attribute. Supply an argument that is in the set and then try the command again.
At C:\Program Files\WindowsPowerShell\Modules\VcRedist\4.1.480\Public\Import-VcIntuneApplication.ps1:79 char:66
+ ...          $RequirementRule = New-IntuneWin32AppRequirementRule @params
+                                                                   ~~~~~~~
    + CategoryInfo          : InvalidData: (:) [New-IntuneWin32AppRequirementRule], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,New-IntuneWin32AppRequirementRule

Add-IntuneWin32App : Cannot validate argument on parameter 'RequirementRule'. The argument is null or empty. Provide an argument that is not
null or empty, and then try the command again.
At C:\Program Files\WindowsPowerShell\Modules\VcRedist\4.1.480\Public\Import-VcIntuneApplication.ps1:114 char:47
+             $Application = Add-IntuneWin32App @Win32AppArgs
+                                               ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Add-IntuneWin32App], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Add-IntuneWin32App

New-IntuneWin32AppRequirementRule : Cannot validate argument on parameter 'MinimumSupportedWindowsRelease'. The argument "1809" does not belong
to the set "W10_1607,W10_1703,W10_1709,W10_1803,W10_1809,W10_1903,W10_1909,W10_2004,W10_20H2,W10_21H1,W10_21H2,W10_22H2,W11_21H2,W11_22H2"
specified by the ValidateSet attribute. Supply an argument that is in the set and then try the command again.
At C:\Program Files\WindowsPowerShell\Modules\VcRedist\4.1.480\Public\Import-VcIntuneApplication.ps1:79 char:66
+ ...          $RequirementRule = New-IntuneWin32AppRequirementRule @params
+                                                                   ~~~~~~~
    + CategoryInfo          : InvalidData: (:) [New-IntuneWin32AppRequirementRule], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,New-IntuneWin32AppRequirementRule

Add-IntuneWin32App : Cannot validate argument on parameter 'RequirementRule'. The argument is null or empty. Provide an argument that is not
null or empty, and then try the command again.
At C:\Program Files\WindowsPowerShell\Modules\VcRedist\4.1.480\Public\Import-VcIntuneApplication.ps1:114 char:47
+             $Application = Add-IntuneWin32App @Win32AppArgs
+                                               ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Add-IntuneWin32App], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Add-IntuneWin32App
```

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Connect-MSIntuneGraph -TenantID tenant.com
2. $Vc = Get-VcList -Release "2022"
3. $VcList = Save-VcRedist -VcList $Vc -Path C:\Temp\VcRedist
4.  Import-VcIntuneApplication -VcList $VcList

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* **VcRedist Version**: 4.1.480
* **PowerShell Version**: 5.1.22621.963
* **Operating System**: Windows 11

## Failure Logs

Please include any relevant log snippets or files here.

**Comment by aaronparker** [comment]
The OS tags have changed recently, so I need to update the template JSON file.

**Comment by aaronparker** [comment]
Should be fixed in [v4.1.483](https://github.com/aaronparker/vcredist/releases/tag/v4.1.483)
